packages/sdk/src/browser-wallets/magiceden/index.ts

```diff
@@ -146,4 +183,10 @@ async function signMessage(
   );
 }
 
-export { getAddresses, isInstalled, signMessage, signPsbt };
+export {
+  getAddresses,
+  getMagicEdenWalletProvider,
```

Comment:
Let's not expose `getMagicEdenWalletProvider` to consumers, can we move it to `magiceden/utils.ts` instead?